src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/Footer.tsx

```diff
@@ -11,8 +12,10 @@ export const Footer: React.FC = () => {
 
   const trackStartSellingClick = () => {
     trackEvent({
-      action: "clickedStartSelling",
-      context_module: "Footer",
+      action: ActionType.tappedConsign,
+      context_module: "Footer", // not importing the name from cohesion as a exeption, should NOT be done in other places
```

Comment:
I'm not sure I understand the reasoning here: couldn't we introduce a new `context_module` in Cohesion that has a name different from "footer" (like "landing_page_footer") or just "footer"?

Reply:
We are already using "Footer" and "Header" for the landing page (the previous version of it). In cohesion we have module names "footer" and "header" - the capital/not-capital letter makes it a completely new event on Audrey's side. We agreed with Audrey that for this page we keep using the module names we already have to keep the code on her side cleaner.

Did I explain the reasoning?

Reply:
And I added the comment so people don't use the same pattern when implementing tracking for other features